tests/integration/__init__.py

```diff
@@ -172,7 +172,7 @@ def _get_dse_version_from_cass(cass_version):
         cassandra_version = Version(mcv_string)
 
     CASSANDRA_VERSION = Version(mcv_string) if mcv_string else cassandra_version
-    CCM_VERSION = cassandra_version if mcv_string else CASSANDRA_VERSION
+    CCM_VERSION = mcv_string or cv_string
```

Comment:
The changes here are to avoid using the packaging.Version with the CCM_VERSION. In this case, `4.0-alpha2` was converted to `4.0a2`

Reply:
Can we use the `mcv_string if mcv_string else cv_string` syntax?

Reply:
yeah, I'm fine if we prefer to use this more explicit syntax. will fix